### PR TITLE
Announcing the new SIP process

### PR DESCRIPTION
**Pull request by heathermiller**
Review by @jvican

### CODE REVIEW

File: blog/_posts/2016-07-12-a-reformed-scala-improvement-process.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+layout: blog
+post-type: blog
+by: Heather Miller
+title: A Reformed Scala Improvement Process
+---
+
+This past May, we held our first [advisory board meeting](http://scala-lang.org/blog/2016/05/30/scala-center-advisory-board.html), to discuss some of the areas that the newly-formed Scala Center should focus on.
+
+One initiative put forward by the members of the committee, and unanimously voted as an initiatve that the Scala Center should _immediately_ undertake was to clarify the governance of Scala, and to in particular focus on ensuring that there is an open, collaborative process in place for evolving the language and standard libraries.
```
Comment (jvican) [inline blog/_posts/2016-07-12-a-reformed-scala-improvement-process.md]: typo: second 'initiatve'

File: blog/_posts/2016-07-12-a-reformed-scala-improvement-process.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+We hope that with the new process will enable a more straightforward path forward for both Scala core maintainers, and current or prospective SIP authors interested in collaborating on evolving the language and standard libraries.
+
+Importantly, the new SIP process creates a _SIP process lead_ role, who's responsible for enforcing and shaping these guidelines _in the broader interest of all Scala users/contributors_. To fill that role, the Scala Center is happy to announce that we've hired [Jorge Vicente Cantero](github.com/jvican). So far, Jorge is responsible for having collected community feedback on the current process, and for proposing the new and improved SIP process. Jorge will be taking over the reigns from the capable hands of Seth Tisue and previously Dick Wall, who created the SLIP process and spearheaded the SIP/SLIP process throughout much of 2015. We're immensely grateful to Dick for kicking off this effort, and hope to make him proud with our additions in the future!
```
Comment (jvican) [inline blog/_posts/2016-07-12-a-reformed-scala-improvement-process.md]: Broken link: should be 'http://github.com/jvican'. It doesn't work for me at least.

**Comment by jvican** [comment]
LGTM

File: blog/_posts/2016-07-12-a-reformed-scala-improvement-process.md (hunk 1 above)
Comment (heathermiller) [inline blog/_posts/2016-07-12-a-reformed-scala-improvement-process.md]: Fixed!

**Comment by heathermiller** [comment]
Hmm, missing commits..

**Comment by heathermiller** [comment]
...

**Comment by heathermiller** [comment]
.

**Pull request merged**